src/main/java/uk/gov/hmcts/darts/automation/utils/TestData.java

```diff
@@ -0,0 +1,276 @@
+package uk.gov.hmcts.darts.automation.utils;
+
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+import java.util.Calendar;
+import java.util.Arrays;
+import java.util.Properties;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.Year;
+import java.time.format.DateTimeFormatter;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.junit.jupiter.api.Assertions;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import org.junit.jupiter.api.Test;
+
+
+public class TestData {
+	private static Logger log = LogManager.getLogger("TestData");
+	static String parameterFileName = "src/test/resources/testData.properties";
+	
+	static String currentYear =  Year.now().toString();
+	Properties propFile; // = new Properties();
+	Properties properties; // = new Properties();
+	public String statusCode = "";
+	public String responseString = "";
+	
+	public TestData() {
+		propFile = new Properties();
+		properties = new Properties();
```

Comment:
Did I miss it in the method or is this dead code now?

Reply:
looks good to me